// mocker/mocker.js
/**
 * 注意保持此文件的干净，避免git冲突
 * 每次发布到线下的时候删除自己的配置
 */
const proxy = {
  'GET /mock/user': {
    id: 1,
    username: 'kenny',
    sex: 6,
    nick: '芬达六块腹肌的'
  },
  'GET /mock/user/list': [
    {
      id: 1,
      username: 'kenny',
      sex: 6
    },
    {
      id: 2,
      username: 'kenny',
      sex: 6
    }
  ],
  'POST /mock/login/account': (req, res) => {
    const { password, username } = req.body;
    if (password === '888888' && username === 'admin') {
      return res.send({
        status: 'ok',
        code: 0,
        token: 'sdfsdfsdfdsf',
        data: {
          id: 1,
          username: 'kenny',
          sex: 6
        }
      });
    }
    return res.send({
      status: 'error',
      code: 403
    });
  },
  'DELETE /mock/user/:id': (req, res) => {
    console.log('---->', req.body);
    console.log('---->', req.params.id);
    res.send({ status: 'ok', message: '删除成功！' });
  },

  // ========================== 以上为mock数据示例,下面是自己的mock数据 start
  // 提交代码遇到冲突可删除

  // ========================== 自己的mock数据 end
};

module.exports = proxy;
